cmake/README.md

```diff
@@ -9,7 +9,7 @@ The following variables are provided for oneDPL configuration:
 | Variable                     | Type   | Description                                                                                   | Default value |
 |------------------------------|--------|-----------------------------------------------------------------------------------------------|---------------|
 | ONEDPL_USE_PARALLEL_POLICIES | BOOL   | Enable parallel policies                                                                      | ON            |
-| ONEDPL_BACKEND               | STRING | Threading backend; supported values: tbb, sycl, sycl_only, ...; the default value is defined by compiler: sycl for DPC++ and tbb for others | tbb/sycl      |
+| ONEDPL_BACKEND               | STRING | Threading backend; supported values: tbb, dpcpp, dpcpp_only, ...; the default value is defined by compiler: dpcpp for DPC++ and tbb for others | tbb/dpcpp |
```

Comment:
`tbb, dpcpp, dpcpp_only, serial;`

Reply:
serial is not handled by build system yet, it should be added separately